lms/urls.py

```diff
@@ -631,6 +646,23 @@
         'student_profile.views.learner_profile',
         name='learner_profile',
     ),
+        # Student performance
+    url(
+        r'^/performance/'.format(
+            username_pattern=settings.USERNAME_PATTERN,
+        ),
+        'student_profile.views.learner_performance',
+        name='learner_performance',
+    ),
+    # Student profile_statistics
+    # TODO remover the user pattern. Security. Get user from session. As everywhere else. 
```

Comment:
I guess this TODO should be removed as a user from the session is current, or viewing, user, while {username} is the user being viewed, right? (as we discussed)